studio/next.config.mjs

```diff
@@ -81,4 +100,60 @@ const config = {
   },
 };
 
-export default withMarkdoc({ mode: "static" })(config);
+const withOptionalSentryConfig = (org, project, config) =>
```

Comment:
What impact does Sentry have on the page sizes / bundle?

Reply:
on main:
```
132M    .next

7.4M    .next/static
6.9M    .next/static/chunks
1.5M    .next/static/chunks/pages

14M     .next/server
13M     .next/server/pages
14M     .next/standalone/studio/
```
on the branch:
```
221M    .next

8.4M    .next/static
7.3M    .next/static/chunks
1.9M    .next/static/chunks/pages

58M     .next/server
25M     .next/server/pages
58M     .next/standalone/studio
```
on the branch with reactComponentAnnotation disabled:
```
220M    .next/

8.2M    .next/static
7.1M    .next/static/chunks
1.7M    .next/static/chunks/pages

58M     .next/server
25M     .next/server/pages
58M     .next/standalone/studio
```
For the static builds it is approx 1mb. 
For server and standalone things increased approx 4 times

Reply:
We should do more tests to see how it affects UX. From your numbers This looks significant.

Reply:
The source maps are doing a huge part here especially for server and standalone.
Without the source maps, which are deleted after uploading them to sentry, it ends up with about 300kb.
```
# on branch
7.7M        .next/static

115M        .next/standalone
22M         .next/server
```

```
# on main
7.4M        .next/static

107M        .next/standalone
14M         .next/server
```

Session Replay brings in a lot:
```
7.9M .next/static
```

What about disabling session replay in the first place?

If the outputs we get from Sentry is already giving us some insights about that something is broken, the session replay could be temporarily enabled when hard to reproduce bugs have been reported.

Reply:
Ok, that makes sense. Sourcemaps are only generated when sentry is enabled right?

Reply:
Yes, the handling for sentry is part of the nextjs plugin configuration:
```                                                                                                          
    org: org,                                                                                                
    project: project,                                                                                        
    sourcemaps: {                                                                                            
      disable: true,                                                                                         
      deleteSourcemapsAfterUpload: true,                                                                     
    },                                                                                                       
```
It's currently hardcoded I don't see, that we would not use it at the moment.

Reply:
Ok, thanks